.gitlab-ci.yml

```diff
@@ -32,6 +32,7 @@ variables:
   MENDER_CONNECT_REV: "master"
   MENDER_BINARY_DELTA_VERSION: "latest"
   DEVICECONFIG_REV: "master"
+  MENDER_CONFIGURE_MODULE_REV: "master"
```

Comment:
As it stands, we don't actually need to clone mender-configure at all, do we? Since it is independent, it will just be fetched by the normal Yocto mechanism, and integration tests for it will not run here. We could keep this variable so that the version is configurable, but I would imagine it would work more like the `MENDER_BINARY_DELTA_VERSION`, two lines above (actually it should be space separated from the rest, since its semantics are different, e.g. `VERSION != REV`). The rest of the code below would then not be needed.

This is assuming that we stick to the idea of the mender-configure module being completely independent. I think I still favor that approach, but I'm not married to the idea, so I'd like your opinion on it.

Reply:
TLDR; you are completely right. I reworked it now to just allow setting an specific version (in case we need it down the road)

I originally didn't foresee required changes in `mender-qa`, but my first build of the other PR triggered a yocto build error that [now I realized that] I miss-interpreted, making me think that I had to specify a PREFERRED_VERSION in `mender-qa`.

But yeah, we should not and we do not want to handle this version together with the rest, so I modified it to use `MENDER_CONFIGURE_MODULE_VERSION`, separated from the rest, and added just in case logic.

> This is assuming that we stick to the idea of the mender-configure module being completely independent. I think I still favor that approach, but I'm not married to the idea, so I'd like your opinion on it.

I am not married to the idea neither, but I think it deserves a chance!